components/kubeflow/kfserving/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.6-slim
 
-RUN pip install kubernetes==9.0.0 kfserving==0.1.1 requests==2.22.0
+RUN pip install kubernetes==9.0.0 kfserving==0.1.1 requests==2.22.0 Flask==1.1.1 flask-cors==3.0.8
```

Comment:
Nit. It's a good habit to use `pip3`, since some images have `pip` pointing to python2.

Reply:
Thanks. Done.